testing/adapter/index.js

```diff
@@ -7,6 +7,21 @@ const {
 
 const { exit } = require('process')
 
+function mintErc20 ({ ethAccountAddress, amount, ethNodeUrl, ethErc20Address, ethErc20AbiPath }) {
+  const robustWeb3 = new RobustWeb3(ethNodeUrl)
+  const web3 = robustWeb3.web3
+  const ethContract = new web3.eth.Contract(
+    JSON.parse(fs.readFileSync(ethErc20AbiPath)),
+    remove0x(ethErc20Address)
+  )
+  ethContract.methods
+    .mint(ethAccountAddress, Number(amount))
+    .send({ from: ethAccountAddress, gas: 5000000 }).then(result => {
+      console.log('OK')
+      exit(0)
+    })
+}
```

Comment:
FYI, this is how we should be able to rewrite this function in async style (it does not matter in this case):

```suggestion
async function mintErc20 ({ ethAccountAddress, amount, ethNodeUrl, ethErc20Address, ethErc20AbiPath }) {
  const robustWeb3 = new RobustWeb3(ethNodeUrl)
  const web3 = robustWeb3.web3
  const ethContract = new web3.eth.Contract(
    JSON.parse(fs.readFileSync(ethErc20AbiPath)),
    remove0x(ethErc20Address)
  )
  const result = await ethContract.methods
    .mint(ethAccountAddress, Number(amount))
    .send({ from: ethAccountAddress, gas: 5000000 });
  console.log('OK')
  exit(0)
}
```